modules/javafx.graphics/src/main/java/javafx/stage/Window.java

```diff
@@ -1117,6 +1117,9 @@ public final ObjectProperty<EventHandler<WindowEvent>> onHiddenProperty() {
                                                            0, 0);
                     }
 
+                    // Give subclasses a chance to adjust the window bounds
```

Comment:
Unrelated to this Enhancement, but I wonder if the `centerOnScreen` line before could also be rewritten using this system (at one point). Now, it looks like he might be centering the `Window`, and then fix the bounds to something else, wasting some time.